app/src/main/java/com/github/featuredetectandroid/utils/conversions/LuminanceArrayToRGB.kt

```diff
@@ -0,0 +1,7 @@
+package com.github.featuredetectandroid.utils.conversions
+
+private const val NUMBER_OF_COMPONENTS = 3
+
+fun luminanceArrayToRGB(luminanceByteArray: ByteArray) = ByteArray(
```

Comment:
Let's put this and `luminanceArrayToBitmap` into a single file and name it something like `LuminanceConversions`

Reply:
fixed